pages/admin/form-builder.tsx

```diff
@@ -21,16 +20,7 @@ const Welcome: React.FC<WelcomeProps> = () => {
   );
 };
 
-export const getServerSideProps = requireAuthentication(async ({ user, locale }) => {
-  const userAbility = user?.privileges && createAbility(user.privileges);
-  if (userAbility?.cannot("update", "FormRecord")) {
-    return {
-      redirect: {
-        destination: `/${locale}/admin/unauthorized/`,
-        permanent: false,
-      },
-    };
-  }
+export const getServerSideProps = requireAuthentication(async ({ user: { ability }, locale }) => {
```

Comment:
`ability` is detected as not being used. Did you just add it for the future?

Reply:
weird... I had added a line checking for a rule but it seems to have disappeared.  
Good catch